src/sentry/models/groupowner.py

```diff
@@ -63,17 +80,82 @@ def owner(self):
         return ActorTuple.from_actor_identifier(self.owner_id())
 
 
-def get_owner_details(group_list):
+def get_owner_details(group_list: List[Group]) -> List[OwnersSerialized]:
     group_ids = [g.id for g in group_list]
     group_owners = GroupOwner.objects.filter(group__in=group_ids)
     owner_details = defaultdict(list)
     for go in group_owners:
         owner_details[go.group_id].append(
-            {
-                "type": GROUP_OWNER_TYPE[GroupOwnerType(go.type)],
-                "owner": go.owner().get_actor_identifier(),
-                "date_added": go.date_added,
-            }
+            OwnersSerialized(
+                type=GROUP_OWNER_TYPE[GroupOwnerType(go.type)],
+                owner=go.owner().get_actor_identifier(),
+                date_added=go.date_added,
+            ),
         )
 
+    org = group_list[0].project.organization if len(group_list) > 0 else None
+    if org and features.has("organizations:release-committer-assignees", org):
```

Comment:
if we could somehow pass the user here as well

Reply:
Done